lib/pandadoc/api/params_validator.rb

```diff
@@ -16,12 +16,14 @@ def initialize(message, parameter)
       class ParameterTypeError < StandardError
         attr_reader :received
         attr_reader :requested
+        attr_reader :parameter
```

Comment:
Thanks for the PR, is there a reason for this additional variable? It looks like you were able to add the `key` to the exception itself. I'm not opposed, would just like to understand the rationale.

Reply:
There is no way of knowing which is the parameter failing the validation. The attr_reader is just to make it explicitly public. What do you mean by 'add the `key` to the exception'?

Reply:
What should we do then?

Reply:
Sorry, will get this merged in soon. Just wanted to make sure `parameter` was actually being used/would be used external to this file as the rest of the change doesn't use it for anything and instead uses the `key` in the messages output. As long as their is justification for it's use it's fine with me.